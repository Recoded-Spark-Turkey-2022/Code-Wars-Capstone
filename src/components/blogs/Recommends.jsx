import React from "react"
import {  useNavigate } from "react-router-dom"
import second from "./2nd.jpg"
import third from "./3rd.jpg"


function Recommends (){
    const navigate = useNavigate();
    const handleBlog2 =(e)=>{
        e.preventDefault();
        navigate(`/blog2`)

    }

    const handleBlog3 =(e)=>{
        e.preventDefault();
        navigate(`/blog3`)
    }
    return(
    <div className="ml-12 w-[75rem] h-[35rem] pt-[10rem] "   >
            <p className=" text-3xl uppercase ">Recommended for you</p>
            <div className="flex space-x-1">
        <button  type="submit" onClick={handleBlog2} className="pt-10 rounded  w-[70rem] h-[35rem]"  to="blog2"> <img src={second} alt="second"  />  </button>
        <button type="submit" onClick={handleBlog3} className="pt-10 rounded w-[70rem] h-[35rem]"  to="blog3"> <img src={third} alt="third"/>  </button>
        </div>
        </div>
    )
}
export default Recommends   
