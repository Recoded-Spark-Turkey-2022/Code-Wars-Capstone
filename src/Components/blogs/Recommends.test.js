
import React from "react";
import renderer from "react-test-renderer";
import Recommends from './Recommends';
import { MemoryRouter } from "react-router-dom";

it("Recommends renders correctly", () => {
    const test = renderer
        .create(
            <MemoryRouter>
                <Recommends />
            </MemoryRouter>
        )
        .toJSON();
    expect(test).toMatchSnapshot();
});