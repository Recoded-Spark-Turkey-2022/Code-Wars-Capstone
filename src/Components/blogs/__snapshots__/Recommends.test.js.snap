// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Recommends renders correctly 1`] = `
<div
  className="xl:w-[75rem] xl:h-[35rem] sm:w-[40rem] sm:h-[20rem] xl:pt-[10rem] sm:pt-[5rem] "
>
  <p
    className=" 2xl:text-3xl sm:text-3xl uppercase "
  >
    Recommended for you
  </p>
  <div
    className="xl:flex space-x-4 xl:pt-[3rem]"
  >
    <button
      className=" rounded  xl:w-[27rem] xl:h-[15rem] sm:w-[20rem] sm:h-[15rem] "
      onClick={[Function]}
      type="submit"
    >
       
      <img
        alt="second"
        src="1st.jpg"
      />
        
    </button>
    <button
      className=" rounded xl:w-[27rem] xl:h-[15rem]  sm:w-[20rem] sm:h-[15rem]"
      onClick={[Function]}
      type="submit"
    >
       
      <img
        alt="third"
        src="2nd.jpg"
      />
        
    </button>
  </div>
</div>
`;
